src/main/java/frc/robot/subsystems/ShooterHoodSubsystem.java

```diff
@@ -60,27 +62,35 @@ public ShooterHoodSubsystem() {
     hoodNEO.setIdleMode(IdleMode.kBrake);
     // The Hood needs to run in the negative direction *towards* the limit switch
     // If it runs the wrong way, flip this invert setting.
-    hoodNEO.setInverted(false);
+    hoodNEO.setInverted(true);
 
     reliableZero = false;
     stopHood();
+
+    CommandScheduler.getInstance().registerSubsystem(this);
```

Comment:
This is handled in the constructor for the supertype SubsystemBase, so you shouldn't do it here.